wordpress-mobile/WordPress-Android
# Issue/uc individual comment moderation

Part of #15101

This PR adds individual comment moderation through Comment Details.

To test:
- Open details of comment and perform moderation action.
- Confirm that moderation and undo functionality are working as expected.
- Confirm that replies to comments appear in the list.

## Regression Notes
None, as it is a feature branch.

PR submission checklist:
